Algorithm.Framework/QCAlgorithmFramework.cs

```diff
@@ -113,9 +114,10 @@ public sealed override void OnFrameworkData(Slice slice)
             // construct portfolio targets from insights
             var targets = PortfolioConstruction.CreateTargets(this, insights);
 
-            // execute on the targets and manage risk
-            Execution.Execute(this, targets);
-            RiskManagement.ManageRisk(this);
+            var riskTargetOverrides = RiskManagement.ManageRisk(this);
+
+            // execute on the targets, overriding targets for symbols w/ risk targets
+            Execution.Execute(this, riskTargetOverrides.Concat(targets).DistinctBy(pt => pt.Symbol));
```

Comment:
Adjusted to reflect the risk module is a post trade risk management; so the targets could feasibly be empty but we'll still have risk adjustments to submit for execution module.